scripts/ci-pull-request.sh

```diff
@@ -15,5 +15,16 @@ source ./scripts/ci-login.sh
 
 ./scripts/build-site.sh preview
 ./scripts/sync-and-test-bucket.sh preview
-./scripts/publish-sentry-release.sh
-./scripts/run-pulumi.sh preview
+
+if [ -n "${SENTRY_AUTH_TOKEN:-}" ]; then
```

Comment:
One thing to note about the intent here -- I deliberately publish to Sentry in order to capture script errors that show up in PR previews (and also post to Slack). So to the extent the aim here is to support "real" previews that could actually be merged, I'd think we'd want to come up with some way to preserve this behavior.

Reply:
Sure that's fine. If we go the route of having a single private mirror, we could set the token and have them be published still.